liri.js

```diff
@@ -0,0 +1,127 @@
+require("dotenv").config();
+var axios = require("axios");
+//var keys = require("./keys.js");
+var fs = require('fs');
+//var Spotify = require('node-spotify-api');
+//var spotify = new Spotify(keys.spotify);
+var moment = require('moment');
+
+
+var action = process.argv[2];
+var userInput = process.argv.slice(3).join(" ");
+
+function bandsInTown() {
+  axios.get("https://rest.bandsintown.com/artists/" + userInput + "/events?app_id=codingbootcamp&date=upcoming").then(
+    function (response) {
+      for (var i = 0; i < 5; i++) {
+        console.log("The venue: " + response.data[i].venue.name);
+        console.log("\n");
+        console.log("the city for the venue is " + response.data[i].venue.city);
+        console.log("\n");
+        console.log(response.data[i].datetime);
+        console.log("\n");
+
+      }
+    }
+  );
+}
+
+function OMDBmovie(){
+axios.get("http://www.omdbapi.com/?t=" + userInput + "&y=&plot=short&apikey=trilogy").then(
+  function (response) {
+    //console.log(response.data.Ratings);
+    console.log("\n");
+    console.log("The Title of the movie is " + response.data.Title);
```

Comment:
You can definitely clean up the code here with not adding all the `console.log("\n")` here and just add them to the actual line too like `console.log("\n The Title of the movie is " + response.data.Title + "\n");`